design-reference/architecture/authn/providers/azuread/integration_test.md

```diff
@@ -0,0 +1,95 @@
+# Purpose/Description
+
+For this design, an engineer building a template would like the ability to test a deployed application with authentication enabled. Currently the Cobalt test-harness cannot successfully complete http calls to an authenticated service and they will return a 401 unauthorized in the response. This is helpful in its self, because it does show that authentication has been configured. However, we would like the ability to also test for a 200 Successful status code. This will give the ability for the CI/CD pipeline to test for both negative and positive results during a deployment.
```

Comment:
Here's that 401 you call out in this description. 100% correct.

![image](https://user-images.githubusercontent.com/10041279/61497096-0d929c00-a984-11e9-8146-e7ecdb615ce0.png)